src/ResourceManager/LocationBasedServices/ChangeLog.md

```diff
@@ -0,0 +1,53 @@
+﻿<!--
+    Please leave this section at the top of the change log.
+
+    Changes for the current release should go under the section titled "Current Release", and should adhere to the following format:
+
+    ## Current Release
+    * Overview of change #1
+        - Additional information about change #1
+    * Overview of change #2
+        - Additional information about change #2
+        - Additional information about change #2
+    * Overview of change #3
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Current Release
+* Integrate with Cognitive Services Management SDK version 2.0.0.
```

Comment:
?? I don;t think this impacts this library

Reply:
you're right.   bad copy/paste from the project i was modeling after.